test/services/LandingServiceSpec.scala

```diff
@@ -227,7 +226,7 @@ class LandingServiceSpec extends AmlsSpec with ScalaFutures with FutureAwaits wi
         service.desConnector.view(any[String], any())(any[HeaderCarrier], any[ExecutionContext], any[Writes[ViewResponse]])
       } thenReturn Future.successful(viewResponse)
 
-      val user = mock[LoggedInUser]
+      //val user = mock[LoggedInUser]
```

Comment:
Commented out. Below lines in same file aswell

Reply:
done